convert2rhel/pkgmanager/handlers/dnf/__init__.py

```diff
@@ -111,6 +111,28 @@ def _enable_repos(self):
                 diagnosis="Loading repository metadata failed with error %s." % (str(e)),
             )
 
+    def _swap_problematic_packages(self):
+        """Swap system packages for their RHEL counterparts in the transaction.
```

Comment:
I'd change to be something like swap base os specific packages

Reply:
@SpyTec "Swap base os specific packages for their RHEL counterparts in the transaction." is fine?

Reply:
Sure! Reflect the function name too

Reply:
@SpyTec Do you like `_swap_base_os_specific_packages()` ?

Reply:
@hosekadam Sure!